src/content/docs/en/guides/middleware.mdx

```diff
@@ -196,6 +197,58 @@ auth response
 validation response
 ```
 
+## Rewriting
+
+<Since v="4.13.0" />
+
+The `APIContext` exposes a method called `rewrite`, which works the same way as [Astro.rewrite](/en/guides/routing/#rewrites).
+
+The `next` function optionally accept a parameter that allows to **rewrite** the current `Request`. For example, if a user isn't logged in, you could **rewrite** the current request without the need of a redirect:
```

Comment:
I removed this part:
> This executes [`context.rewrite()`](/en/reference/api-reference/#contextrewrite) using the provided value (`rewritePayload`)

Because it's incorrect. They are two separate functions with two separate business logic.